understanding/22/consistent-help.html

```diff
@@ -54,11 +54,34 @@ <h2>Intent of Consistent Help</h2>
 
         <p>Without help, some users may abandon the task. They may also fail to correctly complete a task, or they may require assistance from people who do not necessarily keep private information secure.</p>
 
+        <p>It is not the intent of this Success Criterion to require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages. It is also not the intent to require contact information if:</p>
```

Comment:
Since the `While it would be best for all sites to offer at least one of these types of help, this SC does not require help be provided...` is now quite far away and in a separate section, it may be good here to have, before this paragraph about PDF/static docs, a very clear similarly structure paragraph first that says this as part of the actual intent:

`<p>It is not the intent of this Success Criterion to require authors to provide help / access to help. The Criterion only requires that when one of the listed forms of help is available across multiple pages that it be in a consistent location.</p>`

and then slim down the "While it would be best..." paragraph further down - see next comment